ls1intum/Artemis
# `General`: Navigate to the details view after creation or import of text and modeling exercises

### Checklist
#### General


#### Server
#### Client

### Motivation and Context



#5382 added an automated navigation from a lot of entity update views to the detail views after saving them, for example for some exercise types. However, text and modeling exercises currently navigate to the example submissions page after creating or importing them, which is inconsistent.

### Description


Artemis will now navigate to the detail view of text and modeling exercises after creating or importing them.
This PR also removes the creation success alerts like it did for the other entity types as a forward to the details page of a new exercise makes it obvious that it now exists.

### Steps for Testing

Prerequisites:
- 1 Instructor

1. [Create, import] a [text, modeling] exercise
2. Verify that the app forwards you to the details page of that exercise after it was saved
3. Verify that you did **not** get a green success alert in the top right

### Review Progress





#### Code Review
#### Manual Tests
